Frontend/compile-io-frontend-app/src/app/TestServices/assignment.service.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { AssignmentService } from '../services/assignment.service';
+
+describe('AssignmentService', () => {
+  beforeEach(() => TestBed.configureTestingModule({}));
+
+  it('should be created', () => {
```

Comment:
Services are not too difficult to test once you start. Below is an example of what a typical test might look like for this service.
``` typescript
import { TestBed } from '@angular/core/testing';
import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';

import { AssignmentService } from '../services/assignment.service';
import { environment } from 'src/environments/environment';

describe('AssignmentService', () => {
  let service: AssignmentService;
  let http: HttpTestingController;

  beforeEach(() => {
    TestBed.configureTestingModule({
      imports: [HttpClientTestingModule],
      providers: [AssignmentService]
    });
    service = TestBed.get(AssignmentService);
    http = TestBed.get(HttpTestingController);
  });

  it('should get all assignments', done => {
    service.getAssignments('csse120').subscribe(assignments => {
      expect(assignments.length).toBe(1);
      expect(assignments[0].id).toBe(1);
      expect(assignments[0].name).toBe('Intro');
      done();
    });

    const request = http.expectOne(`${environment.BackendapiUrl}/csse120`);

    request.flush([
      {
        id: 1,
        name: 'Intro'
      }
    ]);
  });
});
```
Also, when you get a response back from the API, you generally want the response to be typed instead of the <any> type. This makes for easier testing/debugging.